src/settings.ts

```diff
@@ -581,7 +581,7 @@ export const POSTERPAGE_MAX_VIDEO_PARTICIPANTS = 10;
 
 export const SEARCH_DEBOUNCE_TIME = 200; // ms
 
-export const DEFAULT_DISPLAYED_POSTER_PREVIEW_COUNT = 12;
+export const DEFAULT_DISPLAYED_POSTER_PREVIEW_COUNT = 48;
```

Comment:
@mike-lvov This PR is tiny so will be a quick review. Since you made the `PosterHall` you're probably best at assessing whether we should make this change or not.